Frontend/CO.CDP.OrganisationApp/Pages/Supplier/ConnectedEntity/ConnectedEntityCheckAnswersOrganisation.cshtml.cs

```diff
@@ -210,6 +210,11 @@ private void InitModel(ConnectedEntityState state)
             addresses.Add(AddAddress(state.PostalAddress, CO.CDP.Organisation.WebApiClient.AddressType.Postal));
         }
 
+        if (state.SupplierHasCompanyHouseNumber!.Value == false)
+        {
+            state.CompaniesHouseNumber = null;
+        }
```

Comment:
`SupplierHasCompanyHouseNumber` is nullable - `bool?`

But if it is _actually_ null in reality, then `if (state.SupplierHasCompanyHouseNumber!.Value == false)` would throw a null reference exception.

Simpler to just check like this then we only nullify the  `CompaniesHouseNumber` when actually it's false? If it's null, it simply won't fall into this if statement because `null != false`

```suggestion
        if (state.SupplierHasCompanyHouseNumber == false)
        {
            state.CompaniesHouseNumber = null;
        }
```